src/components/MappingEvents/MappingEventShareBar.js

```diff
@@ -12,10 +29,11 @@ const MappingEventShareBar = ({ className, mappingEvent, buttonCaption, baseUrl,
   const description = mappingEvent.description || mappingEvent.name;
 
   const mailSubject = sharedObjectTitle;
+  const productNameString = productName ? ` on ${productName}` : '';
   // translator: Email text used when sharing a mapping event via email.
   let mailBody = t`Help us out and join the ${
     mappingEvent.name
-  } mapping event on ${productName}. You can find more info here: ${url}`;
+  } mapping event${productNameString}. You can find more info here: ${url}`;
```

Comment:
0: Readd space?

Reply:
The space is now part of the `productNameString` so we don't have an extraneous space when `productName` does not exist.

Reply:
Cool!